plugins/woocommerce-admin/client/task-lists/fills/appearance.js

```diff
@@ -1,431 +1,36 @@
-						</Button>
-					</Fragment>
-				),
-				visible: this.stepVisibility.homepage,
-			},
-			{
-				key: 'logo',
-				label: __( 'Upload a logo', 'woocommerce' ),
-				description: __(
-					'Ensure your store is on-brand by adding your logo',
-					'woocommerce'
-				),
-				content: isPending ? null : (
-					<Fragment>
-						<ImageUpload
-							image={ logo }
-							onChange={ ( image ) =>
-								this.setState( { isDirty: true, logo: image } )
-							}
-						/>
-						<Button
-							disabled={ ! logo && ! isDirty }
-							onClick={ this.updateLogo }
-							isBusy={ isUpdatingLogo }
-							isPrimary
-						>
-							{ __( 'Continue', 'woocommerce' ) }
-						</Button>
-						<Button
-							isTertiary
-							onClick={ () => this.completeStep() }
-						>
-							{ __( 'Skip', 'woocommerce' ) }
-						</Button>
-					</Fragment>
-				),
-				visible: this.stepVisibility.logo,
-			},
-			{
-				key: 'notice',
-				label: __( 'Set a store notice', 'woocommerce' ),
-				description: __(
-					'Optionally display a prominent notice across all pages of your store',
-					'woocommerce'
-				),
-				content: (
-					<Fragment>
-						<TextControl
-							label={ __( 'Store notice text', 'woocommerce' ) }
-							placeholder={ __(
-								'Store notice text',
-								'woocommerce'
-							) }
-							value={ storeNoticeText }
-							onChange={ ( value ) =>
-								this.setState( { storeNoticeText: value } )
-							}
-						/>
-						<Button onClick={ this.updateNotice } isPrimary>
-							{ __( 'Complete task', 'woocommerce' ) }
-						</Button>
-					</Fragment>
-				),
-				visible: true,
-			},
-		];
-
-		return filter( steps, ( step ) => step.visible );
-	}
-
-	render() {
-		const { isPending, stepIndex, isUpdatingLogo, isUpdatingNotice } =
-			this.state;
-		const currentStep = this.getSteps()[ stepIndex ].key;
-
-		return (
-			<div className="woocommerce-task-appearance">
-				<Card className="woocommerce-task-card">
-					<CardBody>
-						<Stepper
-							isPending={
-								isUpdatingNotice || isUpdatingLogo || isPending
-							}
-							isVertical
-							currentStep={ currentStep }
-							steps={ this.getSteps() }
-						/>
-					</CardBody>
-				</Card>
-			</div>
-		);
-	}
-}
-
-const AppearanceWrapper = compose(
-	withSelect( ( select ) => {
-		const { getOption } = select( OPTIONS_STORE_NAME );
-
-		return {
-			demoStoreNotice: getOption( 'woocommerce_demo_store_notice' ),
-		};
-	} ),
-	withDispatch( ( dispatch ) => {
-		const { createNotice } = dispatch( 'core/notices' );
-		const { updateOptions } = dispatch( OPTIONS_STORE_NAME );
-		const { actionTask } = dispatch( ONBOARDING_STORE_NAME );
-
-		return {
-			actionTask,
-			createNotice,
-			updateOptions,
-		};
-	} )
-)( Appearance );
+const useAppearanceClick = () => {
+	const { actionTask } = useDispatch( 'wc/admin/onboarding' );
+	const onClick = () => {
+		actionTask( 'appearance' );
+		window.location = 'themes.php';
```

Comment:
I think it's safer if we build the URL using `getAdminLink` so that there's less chance to break.

Reply:
Good call! I fixed it in [8b150a2].(https://github.com/woocommerce/woocommerce/pull/40239/commits/8b150a2c86a11f9cfe654812d8c1f3a5ec12f775)